Build fails: 'AboutFriendForm::about' will be initialized after

##### Brief Description

OS: Linux Fedora Rawhide
qTox version: 1.12.1
toxcore:  0.1.10
Qt:  5.9.2
…
Reproducible: Always

Build fails on Fedora with the following error:

```
In file included from /builddir/build/BUILD/qTox-1.12.1/src/widget/about/aboutfriendform.cpp:1:0:
/builddir/build/BUILD/qTox-1.12.1/src/widget/about/aboutfriendform.h: In constructor 'AboutFriendForm::AboutFriendForm(QPointer<IAboutFriend>, QWidget*)':
/builddir/build/BUILD/qTox-1.12.1/src/widget/about/aboutfriendform.h:23:28: error: 'AboutFriendForm::about' will be initialized after [-Werror=reorder]
     QPointer<IAboutFriend> about;
                            ^~~~~
/builddir/build/BUILD/qTox-1.12.1/src/widget/about/aboutfriendform.h:22:26: error:   'Ui::AboutFriendForm* AboutFriendForm::ui' [-Werror=reorder]
     Ui::AboutFriendForm* ui;
                          ^~
/builddir/build/BUILD/qTox-1.12.1/src/widget/about/aboutfriendform.cpp:7:1: error:   when initialized here [-Werror=reorder]
 AboutFriendForm::AboutFriendForm(QPointer<IAboutFriend> about, QWidget* parent)
 ^~~~~~~~~~~~~~~
cc1plus: all warnings being treated as errors
make[2]: *** [CMakeFiles/qtox_static.dir/build.make:1710: CMakeFiles/qtox_static.dir/src/widget/about/aboutfriendform.cpp.o] Error 1
```

--- a/src/widget/about/aboutfriendform.cpp
+++ b/src/widget/about/aboutfriendform.cpp
@@ -6,8 +6,8 @@
 
 AboutFriendForm::AboutFriendForm(QPointer<IAboutFriend> about, QWidget* parent)
     : QDialog(parent)
-    , about{about}
     , ui(new Ui::AboutFriendForm)
+    , about{about}
 {
     ui->setupUi(this);
     ui->label_4->hide();

